snippets/ztp_example-sno.yaml

```diff
@@ -76,6 +76,9 @@ spec:
           bootMode: "UEFISecureBoot"
           rootDeviceHints:
             deviceName: "/dev/disk/by-path/pci-0000:01:00.0-scsi-0:2:0:0"
+          # Optionally; This can be used to override the BareMetalHost resource. To avoid the creation or deletion getting stuck, set the `argocd.argoproj.io/sync-wave: "3"` annotation in your override BareMetalHost CR
```

Comment:
We are exposing here a bug about BMH getting stuck. We should not do that. The text could be at Line 64 or 65, something like:
"To override the BareMetalHost CR, set the `argocd.argoproj.io/sync-wave: "3"` annotation in your overriden BareMetalHost CR file"